aptos-move/framework/aptos-experimental/sources/daa_siws_phantom.move

```diff
@@ -0,0 +1,255 @@
+/// Derivable account abstraction that verifies a message signed by
+/// SIWS from a Phantom wallet.
```

Comment:
is this specific to Phantom, or any SIWS should work?

Reply:
➕

Reply:
Right now, we've only verified with Phantom. I'd say we update this comment once we've individually verified each wallet.